src/modules/drawing/components/DrawingInteractions.vue

```diff
@@ -8,28 +8,59 @@ import DrawingMarkerInteraction from '@/modules/drawing/components/DrawingMarker
 import DrawingMeasureInteraction from '@/modules/drawing/components/DrawingMeasureInteraction.vue'
 import DrawingSelectInteraction from '@/modules/drawing/components/DrawingSelectInteraction.vue'
 import DrawingTextInteraction from '@/modules/drawing/components/DrawingTextInteraction.vue'
+import ExtendLineInteraction from '@/modules/drawing/components/ExtendLineInteraction.vue'
+import ExtendMeasureInteraction from '@/modules/drawing/components/ExtendMeasureInteraction.vue'
 import { EditMode } from '@/store/modules/drawing.store'
-
+import log from '@/utils/logging'
 // DOM References
 const selectInteraction = ref(null)
 const currentInteraction = ref(null)
-
 const store = useStore()
 const currentDrawingMode = computed(() => store.state.drawing.mode)
 const editMode = computed(() => store.state.drawing.editingMode)
 
+let selectedLineFeature = null
```

Comment:
you should make a `ref` out of this, as it might change without recreating the whole `useExtendLineInteraction` down the line